code/places-monitor-android/src/phone/java/com/adobe/marketing/mobile/PlacesActivity.java

```diff
@@ -280,5 +275,22 @@ private void onShowRationale(PlacesMonitorLocationPermission locationPermission)
 										  PlacesMonitorConstants.MONITOR_LOCATION_PERMISSION_REQUEST_CODE);
 	}
 
+	/**
+	 * Dispatch an OS event to eventHub, indicating the change in location permission
+	 *
+	 * @param status A {@link String} value representing the status of the changed permission.
```

Comment:
status A is this a typo?

Reply:
ohhh got it might be a hyphen might have helped

Reply:
![Screen Shot 2019-11-14 at 11 07 43 AM](https://user-images.githubusercontent.com/8909148/68888094-1fd0dc80-06cf-11ea-8767-15498f517d68.png)

Reply:
doesn't look too bad on an IDE :) 
I like your idea of having a hyphen. Thought better to stick with javadoc guidelines
* @param ch        the character to be tested
* @param observer  the image observer to be notified